### PR TITLE
add quickstart section

### PR DESCRIPTION
**Pull request by jcantrill**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,16 @@
 An operator to support OKD aggregated cluster logging.  Cluster logging configuration information
 is found in the [configuration](./docs/configuration.md) documentation.
 
+## Quick Start
+To get started with cluster logging and the `cluster-logging-operator`:
+* Ensure Docker is installed on your local system
+```
+$ oc login $CLUSTER -u $ADMIN_USER -p $ADMIN_PASSWD
+$ REMOTE_CLUSTER=true make deploy-example
```
Comment (ewolinetz) [inline README.md]: do you need `REMOTE_CLUSTER` ? Or does this assume that $CLUSTER is a remote cluster

**Comment by ewolinetz** [comment]
question, otherwise lgtm

**Comment by richm** [comment]
ready to merge?

**Comment by jcantrill** [comment]
> ready to merge?

I believe so but I never fully answered https://github.com/openshift/cluster-logging-operator/pull/94/files#r250804985 .  I defer to @richm if there is additional comment?

File: README.md (hunk 1 above)
Comment (richm) [reply README.md]: You need `REMOTE_CLUSTER`

**Pull request merged**